src/modules/commands/discord/population.ts

```diff
@@ -20,10 +21,12 @@ type Result = {
 export default new Command<discord.Message>({
   keyword: 'population',
   description: 'display the current population',
-  help: 'Usage: `{prefix}population` - displays the current population',
+  help: 'Usage:\n `{prefix}population` - displays the current population\n`{prefix}population numbers` - Diplays the current population with numbers',
   alias: ['pop'],
   callback: async ({ args, reply, raw }) => {
-    if (args.length > 0) return
+    validateArgumentRange(args.length, 0, 1)
+
+    const numbersBool = args.length === 1 && args[0] === 'numbers'
```

Comment:
You need to reply with an error when the argument is something else than "numbers".